### PR TITLE
[REVIEW] switch thrust to the NVIDIA github org

### PR DESCRIPTION
**Pull request by rongou**
Thrust recently switched to the NVIDIA github organization.

@harrism @jrhemstad

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
 - PR #538 Upgrade CUB and Thrust to the latest commits
 - PR #542 Pin conda spdlog versions to 1.7.0
 - PR #550 Remove CXX11 ABI handling from CMake
+- PR #578 Switch thrust to use the NVIDIA/thrust repo
```
Comment (harrism) [inline CHANGELOG.md:20]: Why make this so out of order?

**Review by harrism — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rongou) [reply CHANGELOG.md:20]: I just inserted the line randomly to avoid merge conflict. :) Should they be sorted?

**Comment by rongou** [comment]
rerun tests

File: CHANGELOG.md (hunk 1 above)
Comment (harrism) [reply CHANGELOG.md:20]: Seems nicer sorted, but don't worry about it.

**Pull request merged**